Jenkinsfile

```diff
@@ -69,8 +69,8 @@ def buildManifest = {String bitbake_image, String bsp, boolean qtauto ->
 // Run the different jobs in parallel, on different slaves
 parallel 'intel':{
     node("DockerCI") { buildManifest("core-image-pelux-minimal", "intel", false) }
-},'intel-qtauto':{
+}/*,'intel-qtauto':{
```

Comment:
This should go into a different commit.

Reply:
It will be removed completely later on, so that it builds all images.